tests/test_spo_cvsd.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import BOARD, PASSED
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+from scrapy.http import HtmlResponse, Request
+
+from city_scrapers.spiders.spo_cvsd import SpokSvsdSpider
+
+freezer = freeze_time("2024-02-07")
+freezer.start()
+freezer.stop()
+
+# set up response fixtures
+# Because _parse_detail expects meta data, we can't rely on the
+# the file_response helper in this case. We create a response
+# object from scratch.
+test_response = file_response(
+    join(dirname(__file__), "files", "spo_cvsd.json"),
+    url="https://go.boarddocs.com/wa/cvsd/Board.nsf/BD-GetMeetingsList?open&0.320616755508751",  # noqa: E501
+)
+start_date = datetime(2024, 1, 22)
+request_with_meta = Request(
+    url="https://go.boarddocs.com/wa/cvsd/Board.nsf/BD-GetMeeting?open&0.934001628042688",  # noqa: E501
+    meta={"start_date": start_date, "meeting_id": "CYXL4V54CB3F"},
+)
+response_body = open(
+    join(dirname(__file__), "files", "spo_cvsd_detail.json"), "rb"
```

Comment:
Minor thing: should we save the details content in .html file instead of .json?

Reply:
Good catch, @LienDang. Yes, this should be an `html` extension. It's now fixed. Thank you for spotting that 🙏